CHANGES.md

```diff
@@ -1,5 +1,8 @@
 Unreleased
 ----------
+- Add map_workspace_root dune-project stanza to all disabling of
+  mapping of workspace root to /workspace_root. (#6988, fixes #6929, @richardlford)
+  
```

Comment:
```suggestion
- Add map_workspace_root dune-project stanza to all disabling of
  mapping of workspace root to /workspace_root. (#6988, fixes #6929,
  @richardlford)
  
```

Reply:
I had pushed a correction of a typo after you made this suggestion. I'll make your change separately.